### ISSUE TITLE
Redirect from outside on a local ip

### ISSUE THREAD

**Issue by toniolol** [pilotak/docker-rpi-port-forward#1]
Hello,

I think it's not an issue.
I'm using this docker to access to my own router from outside :
sudo docker run -d --name port-forwarding --cap-add=NET_ADMIN -e TARGET_IP=192.168.3.1 -e TARGET_PORT=80 -p 9031:8000 pilotak/port-forwarding

It works from inside (192.168.3.x) but not from outside with NAT rules on my router.

Thanks

**Comment by pilotak** [comment]
Depends how you are trying to access it. If the device where the docker is running is accesible via internet (ie you have public IP address from your internet provider or via VPN through another server in the internet with public IP), you can use this image to access any device in your local network. It won't work otherwise.

**Comment by toniolol** [comment]
Hello.
What i'm trying to do is this :

From outside I'm "calling" my raspberry on port 9031 (using NAT on my router) => This is OK.
REDIRECT have to transmit this call to my Huwaei WiFi local on 192.168.3.1:80 => OK from inside / NOK from outside

My command is this one: 
`sudo docker run -d --name port-forwarding --cap-add=NET_ADMIN -e TARGET_IP=192.168.3.1 -e TARGET_PORT=80 -p 9031:8000 pilotak/port-forwarding`

**Comment by pilotak** [comment]
Sorry i don't understand your explanation. Please tell me the what addresses you are calling from each PC and IPs of your devices (randomize part of the public IP) and add it to the following picture, or draw your own
![int](https://user-images.githubusercontent.com/8101753/153049555-0a9be277-7223-4baf-a1d5-142f4fb009ea.jpg)

**Comment by toniolol** [comment]
Thanks for your help!

All my subnet is on Huawei (192.168.3.1) and I want to access to 192.168.3.1:80 from outside.
My DHCP is provided by Huawei.
My NAT forwarding is provided by Huwaei (plus my Router) and it works perfectly for all my others Dockers.
But Huawei have some protection and don't allow this NAT on himself.

So I imagined to enter my subnet on my RPI (like others dockers I have) and your docker could access the Huawei admin page on port 80.

https://paste.pics/FXU7V

**Comment by pilotak** [comment]
Well picture is worth thousand words.
If you access rpi from the internet lets say you call `48.69.156.56:2000`, you have port `2000` opened on both routers and gets translated to `192.168.3.150:22`. All that needs to be done is to open port ie. `3000` again on both routers and get it translate to `192.168.3.150:9031` this way you can call `48.69.156.56:3000` from outside PC while calling `192.168.3.150:9031` from local PC.

**Comment by toniolol** [comment]
Yes, it works like a charm !
Thanks a lot for your help.
I totally forgot his double translation :(
Thanks for your time and your docker !!!!
